src/DIRAC/Core/Utilities/Extensions.py

```diff
@@ -158,10 +158,13 @@ def _extensionsByPriorityPy3():
   from importlib import metadata  # pylint: disable=no-name-in-module
 
   priorties = defaultdict(list)
-  for entrypoint in set(metadata.entry_points()['dirac']):
-    extensionName = entrypointToExtension(entrypoint)
-    extension_metadata = entrypoint.load()()
-    priorties[extension_metadata["priority"]].append(extensionName)
+  try:
+    for entrypoint in set(metadata.entry_points()['dirac']):
+      extensionName = entrypointToExtension(entrypoint)
+      extension_metadata = entrypoint.load()()
+      priorties[extension_metadata["priority"]].append(extensionName)
+  except KeyError:
```

Comment:
@chrisburr I could not run this "properly" in my installation so that's why this "hack". I didn't give it much thoughts, let me know if it's a bad idea.

Reply:
What was the reason for not being able to?

Reply:
```
 toffo   v7r3-fixes11 → upstream/integration ↑12 {21}  ❲c❳ dirac-development  ~/Devs/DIRACGrid/DIRAC  ipy
Python 3.9.4 (default, Apr  9 2021, 16:34:09) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.24.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from importlib import metadata

In [2]: metadata.entry_points()['dirac']
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-2-7dcc5536b8e9> in <module>
----> 1 metadata.entry_points()['dirac']

KeyError: 'dirac'

```

can't say more

Reply:
I mean why can't you install DIRAC with `pip`? Something like `pip install -e .` from the root of the repository

Reply:
OK sorted. I am pretty sure I had done already, but whatever. I have just re-issued `pip install -e` and works fine. I remove the hack above.

Reply:
Reverted. The tests should be all green now, this would fix the integration branch.